### DISCUSSION TITLE
What is fillament

### DISCUSSION THREAD

**Discussion by prachitiwari-kombee** [filamentphp/filament#13955] [General]

**Comment by leandrocfe** [comment]
I would say: Filament is a powerful collection of full-stack components built on the TALL Stack—Tailwind, AlpineJS, Laravel, and Livewire. These components streamline your application development process by handling the heavy lifting, so you don’t have to. 
With Filament, you can focus on what matters most: your business, while Filament takes care of the rest ;)
